docs/arc42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,74 @@
+| 2        | Provide the infrastructure such that other researchers ("maintainers") can easily setup their own instance. |
+
+## Requirements Overview
+
+| Id  | Requirement                             |                                                                                        |
+|-----|-----------------------------------------|----------------------------------------------------------------------------------------|
+| F1  | Create an instance for a given pathogen | Create an instance of the whole system by given configuration for different pathogens. |
+| F2  | Load data                               | Load sequence data that has to be provided in a defined format.                        |
+| F3  | Store data                              | Store data in compressed form.                                                         |
+| F4  | Provide web access to data              | Provide endpoints for custom user queries to the data.                                 |
+| F5  | Provide statistics                      | Provide monitoring and statistics of usage of the system.                              |
+
+## Quality Goals
+
+| Quality Category       | Quality              | Description                                                                                                       | Id   |
+|------------------------|----------------------|-------------------------------------------------------------------------------------------------------------------|------|
+| Usability              | Ease of use          | Ease of use for the user to hand in queries.                                                                      | Q:U1 |
+|                        | Ease of use          | Ease of use for maintainers to create a new instance for a new pathogen.                                          | Q:U2 |
+|                        | Ease of learning     | The queries should be as easy to write as possible. We provide material to assist in learning the query language. | Q:U3 |
+|                        |                      |                                                                                                                   | Q:U4 |
+| Performance efficiency | Time behaviour       | It is possible to query millions of sequences in less than a second.                                              | Q:P1 |
+|                        | Resource utilization | How many do we have?                                                                                              | Q:P2 |
+|                        | Workload             | How many requests should be tolerable?                                                                            | Q:P3 |
+|                        | Scalability          | How many sequences do we expect?                                                                                  | Q:P4 |
```

Comment:
It is hard to quantify the qualities exactly and it differs for different instances/use cases. Regarding scalability, I'd say that the "performance" (query response time, memory requirement) should grow at most linearly.